app/views/home/_navbar.html.erb

```diff
@@ -1,18 +1,2 @@
-<div id="vertical-nav" class="nav-pf-vertical
-  nav-pf-vertical-with-sub-menus
-  nav-pf-persistent-secondary">
-  <ul class="list-group">
-    <%= render_vertical_menu :top_menu %>
-
-    <% if Setting[:lab_features] %>
-      <%= render_vertical_menu :labs_menu %>
-    <% end %>
-
-    <%= render_vertical_menu :admin_menu %>
-
-    <%= render "home/vertical_taxonomies", :taxonomy_type => "organization", :icon_name => "fa fa-building" if SETTINGS[:organizations_enabled]  %>
-    <%= render "home/vertical_taxonomies", :taxonomy_type => "location", :icon_name => "fa fa-globe" if SETTINGS[:locations_enabled]  %>
-
-    <%= render_vertical_menu :side_menu, true %>
-  </ul>
-</div>
\ No newline at end of file
+<div id="layout"></div>
+    <%= mount_layout %>
```

Comment:
nitpick: extra whitespace. in fact, lets just remove this file? (just include this two lines in layouts/base.html.erb) and i guess you can remove the views/home path as well.